pyartcd/pyartcd/pipelines/ocp4_konflux.py

```
@@ -878,44 +884,247 @@ def trigger_bundle_build(self):
+        if self.skip_rhcos_integration_tests:
+            return
+
+        from pyartcd.rhcos_jenkins_client import RhcosJenkinsClient
+
+        try:
+            client = RhcosJenkinsClient(kubeconfig_env_var='RHCOS_JENKINS_KUBECONFIG')
+            # Resolve and cache the token now, while the kubeconfig file still exists
+            client.retrieve_auth_token()
+            self.rhcos_jenkins_client = client
+            LOGGER.info("RHCOS Jenkins client authenticated; token cached for later integration tests")
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)